ReturnBookControl.java

```diff
@@ -1,76 +1,76 @@
 public class ReturnBookControl {
 
-	private ReturnBookUI Ui;
-	private enum CONTROL_STATE { INITIALISED, READY, INSPECTING };
-	private CONTROL_STATE sTaTe;
+	private ReturnBookUI returnBookUi;
+	private enum RetunrBookControlState { INITIALISED, READY, INSPECTING };

```

Comment:
RetunrBookControlState into ReturnBookControlState just a spelling mistake @GoJoeyGo thankyou

Reply:
happy to change

Reply:
thanks